### PR TITLE
Replace all whitespace by nothing in failing tests

### LINKED ISSUE #25: Tests are failing because not all whitespace is removed in some expected values
\t, \n are not removed in the expected values

### PR DESCRIPTION
**Pull request by jaapz**
Fixes #25 

I am not entirely sure I am convinced of the use of these fuzzing tests. To me it looks like we are just re-implementing the camelize function for each fuzzed tests, to get our "expected" value. 

Maybe we can replace these tests by parametrized tests where we define an input and expected value in a hardcoded list. That does seem to make more sense to me, at least.

But at least this fixes the failing tests.

### CODE REVIEW

File: tests/CamelizeTest.elm [hunk 1]
```diff
@@ -27,8 +28,11 @@ camelizeTest =
         , fuzz string "It is the same lowercased string after removing the dashes and spaces" <|
             \s ->
                 let
-                    expected =
-                        replace "-" "" >> replace "_" "" >> replace " " "" >> String.toLower
+                    expected = replace "-" ""
+                        >> replace "_" ""
+                        >> replace " " ""
```
Comment (lorenzo) [inline tests/CamelizeTest.elm]: I think this line can be removed

File: tests/HumanizeTest.elm [hunk 2]
```diff
@@ -60,6 +60,7 @@ humanizeTest =
                         replace " " ""
```
Comment (lorenzo) [inline tests/HumanizeTest.elm]: This line can now be removed

**Comment by lorenzo** [comment]
> I am not entirely sure I am convinced of the use of these fuzzing tests.

The value is to test properties of the implementation. The way you do it is by removing "rules" and testing simpler rules of the solution to make sure that they still hold.

Property testing and unit testing are not mutually exclusive, we can have both.

**Comment by jaapz** [comment]
Maybe instead of testing explicitly for an "expected" value, we should test for the properties of the values returned by the function? So instead of re-implementing the capitalize function in our fuzz tests, we can do several assertions on the resulting value, like "contains no spaces", "first letter is uppercase". That would make much more sense to me, at least.

**Comment by lorenzo** [comment]
The tests are not entirely implementing the functions, they are actually testing properties. If you have a smarter way of testing the properties I would love to learn!

**Comment by sporto** [comment]
That test mirrors the implementation, I don't see how is that a good test. I'm haven't use fuzz testing much, but there has to be a better approach.

**Comment by sporto** [comment]
Hmm, don't mind my comment above, I'm not sure what I'm talking about. It just looks like the test is doing too much.

**Pull request merged**